source/sass/main.scss

```diff
@@ -391,7 +391,7 @@ button {
 
     h4 {
       color: #fff;
-      padding: 23px 0;
+      margin: 1.45rem 0;
```

Comment:
I had to change this, because the row was getting in the way of a collapsing margins, causing the height of the headers to be taller on the bottom than the top. It was showing an extra margin on the bottom caused by the h4 settings. My override for padding no longer made much sense, and figured just set it via margin, and thus override the default h4 margin-bottom.